### PR TITLE
Extraline

### PR DESCRIPTION
**Pull request by sedin-asvin**
just a demo pull request from develoepr side

### CODE REVIEW

File: calculation.rb [hunk 1]
```diff
@@ -22,4 +22,4 @@ def div(a, b)
   puts "Sub result: " + calc.sub(5, 2).to_s
```
Comment (javajargon) [inline calculation.rb:22]: This line you can remove

Comment (sedin-asvin) [reply calculation.rb:22]: sure

Comment (javajargon) [inline calculation.rb:22]: hope u will remove

**Review by sedin-asvin — COMMENTED** [review]
definitely

**Review by javajargon — APPROVED** [review]

**Comment by javajargon** [comment]
Thank you your code will be reviwed

**Comment by javajargon** [comment]
i think your code looks fine

**Pull request merged**